composer/helpers/v2/src/UpdateChecker.php

```diff
@@ -14,7 +14,7 @@ final class UpdateChecker
 {
     public static function getLatestResolvableVersion(array $args): ?string
     {
-        [$workingDirectory, $dependencyName, $gitCredentials, $registryCredentials] = $args;
+        [$workingDirectory, $dependencyName, $gitCredentials, $registryCredentials, $latestAllowableVersion] = $args;
```

Comment:
You seem to have a merge issue here, these changes shouldn't be here

Reply:
No, that is what was in main as of June 12.